src/e2e/step-definitions/external-systems.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+import { Then, When } from "@cucumber/cucumber";
+import { Page } from "@playwright/test";
+import { CustomWorld } from "../support/worlds/world";
+
+const ONE_MINUTE_IN_MS = 60_000;
+
+// Reference to smartdocuments page
+let smartDocumentsPage: Page;
+
+When(
+  "Employee {string} clicks on Create Document for zaak",
+  { timeout: ONE_MINUTE_IN_MS },
+  async function (this: CustomWorld, user) {
+    await this.page.getByText("note_addDocument maken").click();
+
+    const sidebar = this.page.locator("div.sidenav-title");
+    await sidebar.waitFor({ state: "visible" });
+    await sidebar.getByText("Document maken");
+  },
+);
+
+When(
+  "Employee {string} fills in the create document form",
+  { timeout: ONE_MINUTE_IN_MS },
+  async function (this: CustomWorld, user) {
+    const submitButton = this.page.locator("#opslaan_button");
+    await submitButton.waitFor({ state: "visible" });
+
+    // Verify that the button is initially disabled
```

Comment:
Same here. I would remove this comment. The problem with code comments is that often at some point in time they grow out of sync with the code so I would only add them if it really adds value. Ideally code should be self-explanatory. :-)

Reply:
✅